components/raftstore/src/store/peer_storage.rs

```diff
@@ -1385,48 +1385,21 @@ where
 
     pub fn compact_to(&mut self, idx: u64) {
         self.compact_cache_to(idx);
-
         self.cancel_generating_snap(Some(idx));
     }
 
     pub fn compact_cache_to(&mut self, idx: u64) {
         self.cache.compact_to(idx);
-        let rid = self.get_region_id();
-        if self.engines.raft.has_builtin_entry_cache() {
-            self.engines.raft.gc_entry_cache(rid, idx);
-        }
```

Comment:
why deleting this part? (e.g. what if raft uses rocksdb?)

Reply:
Neither raft engine nor rocksdb has builtin entry cache now, it's deprecated.